src/utils/googleTagManager.ts

```diff
@@ -76,21 +80,17 @@ export const loadGoogleTagManager = (): void => {
   // Cache name to prevent tracking of same page
   currentPathname = history.location.pathname
 
-  const page = getAnonymizedLocation()
-
+  const page_path = getAnonymizedPathname()
   TagManager.initialize({
     gtmId: GOOGLE_TAG_MANAGER_ID,
     ...GTM_ENVIRONMENT,
     dataLayer: {
       // Must emit (custom) event in order to trigger page tracking
       event: GTM_EVENT.PAGEVIEW,
       chainId: _getChainId(),
-      page,
-      // Allow only GA4 configuration and GA4 custom event tags
-      // @see https://developers.google.com/tag-platform/tag-manager/web/restrict
-      'gtm.allowlist': ['gaawc', 'gaawe'],
-      // Block JS variables and custom scripts
-      'gtm.blocklist': ['j', 'jsm', 'customScripts'],
```

Comment:
Can we restore the blocklist too?

Reply:
Worked as well 👍  so it was just missing the Custom Events in the allowlist.

Reply:
Oh, dang. That explains why the events dropped 🙈